godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/Rect2.kt

```diff
@@ -0,0 +1,263 @@
+            ((rect.position.x + rect.size.x) < (position.x + size.x)) &&
+            ((rect.position.y + rect.size.y) < (position.y + size.y))
+    }
+
+    /**
+     * Returns this Rect2 expanded to include a given point.
+     */
+    fun expand(vector: Vector2): Rect2 {
+        val r = Rect2(this.position, this.size)
+        r.expandTo(vector)
+        return r
+    }
+
+
+    internal fun expandTo(vector: Vector2) {
+        val begin = position
+        val end = position + size
+
+        if (vector.x < begin.x) {
+            begin.x = vector.x
+        }
+        if (vector.y < begin.y) {
+            begin.y = vector.y
+        }
+        if (vector.x > end.x) {
+            end.x = vector.x
+        }
+        if (vector.y > end.y) {
+            end.y = vector.y
+        }
+
+        position = begin
+        size = end - begin
+    }
+
+    /**
+     * Returns the area of the Rect2.
+     */
+    fun getArea(): RealT {
+        return size.x * size.y
+    }
+
+    /**
+     * Returns a copy of the Rect2 grown a given amount of units towards all the sides.
+     */
+    fun grow(by: RealT): Rect2 {
+        val g = Rect2(this.position, this.size)
+        g.position.x -= by
+        g.position.y -= by
+        g.size.x += by * 2
+        g.size.y += by * 2
+        return g
+    }
+
+    /**
+     * Returns a copy of the Rect2 grown a given amount of units towards all the sides.
+     */
+    fun growIndividual (left: RealT, top: RealT,right: RealT,bottom: RealT ): Rect2 {
+        val g = Rect2(this.position, this.size)
+        g.position.x -= left
+        g.position.y -= top
+        g.size.x += left + right
+        g.size.y += top + bottom
+        return g
+    }
+
+    /**
+     * Returns a copy of the Rect2 grown a given amount of units towards all the sides.
+     */
+    fun growMargin (margin: Margin, by: RealT ): Rect2 {
+        val g = Rect2(this.position, this.size)
+        g.position.x -= by
+        g.position.y -= by
+        g.size.x += by * 2
+        g.size.y += by * 2
+        return g
+    }
+
+    /**
+     * Returns true if the Rect2 is flat or empty.
+     */
+    fun hasNoArea(): Boolean {
+        return size.x <= 0 || size.y <= 0
+    }
+
+    /**
+     * Returns true if the Rect2 contains a point.
+     */
+    fun hasPoint(point: Vector2): Boolean {
+        return when {
+            point.x < position.x                -> false
+            point.y < position.y                -> false
+            point.x >= (position.x + size.x)    -> false
+            point.y >= (position.y + size.y)    -> false
+            else                                -> true
+        }
+    }
+
+    /**
+     * Returns true if the Rect2 overlaps with b (i.e. they have at least one point in common).
+     * If include_borders is true, they will also be considered overlapping if their borders touch, even without intersection.
+     */
+    fun intersects(rect: Rect2): Boolean {
```

Comment:
Either the comment is not correct or a param is missing but there is no param boolean called `include_borders`

Reply:
Yeah sorry, I forgot to add it.